### PR TITLE
Fix bugs with multiclass model

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

File: cpp/oneapi/dal/algo/svm/backend/gpu/train_cls_kernel_thunder_dpc.cpp [hunk 1]
```diff
@@ -93,6 +93,10 @@ static result_t call_daal_kernel(const context_gpu& ctx,
                                                                             *daal_labels,
                                                                             daal_model.get(),
                                                                             daal_svm_parameter));
+    const std::int64_t n_sv = daal_model->getSupportIndices()->getNumberOfRows();
+    if (n_sv == 0) {
+        return train_result<Task>{};
```
Comment (owerbat) [inline cpp/oneapi/dal/algo/svm/backend/gpu/train_cls_kernel_thunder_dpc.cpp:98]: You need to add Task template to `train()` and `call_daal_kernel()`

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Comment by PetrovKP** [comment]
/intelci: run
daal4py_repo=https://github.com/PetrovKP/daal4py.git
daal4py_branch=support-empty-table

**Pull request merged**